### ISSUE TITLE
Error installing VSCode on Microsoft Surface Pro 3

### ISSUE THREAD

**Issue by shadowkyros** [Microsoft/vscode#13082]
- VSCode Version: 1.5 and Insiders build (tried them both)
- OS Version: Windows 10 Pro 
Microsoft Surface Pro 3 i7, 256gb hard drive, 8gb RAM (up to date on all Windows Updates as of 09/28/16).

Problem: I can the program running and working on my computer perfectly fine until about 3 weeks ago it stopped working. I uninstalled, deleted config files, reinstalled and it still wouldn't work. I installed the Insiders Build and it worked for a few days but then it too stopped working. Now, if I installed the program and go through the installation it appears that everything is working fine. However, when I go to launch the program, it merely tries to open it but then nothing happens. No error message or issue pops up. I've tried running the program as an Administrator and under compatibility mode but nothing seems to work. Can you please help me figure out why this software won't run on my computer?

**Comment by shadowkyros** [comment]
Update:  Used the terminal to get this information when it tries to load. Hopefully this helps the issue. I'm going to try and uninstall and/or reinstall python so that isn't an issue. 
Microsoft Windows [Version 10.0.14393]
(c) 2016 Microsoft Corporation. All rights reserved.

C:\Users\Randall>code . --verbose --disable-extensions

Conflicting configuration setting: python.pythonPath at python with "python2"
TypeError: Cannot create property 'pythonPath' on string 'python2'
    at o (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:15848)
    at Object.f [as flatten] (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:17337)
    at e.getConsolidatedConfig (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:19636)
    at e.getConfiguration (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:19287)
    at new e (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:11:6674)
    at v (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:4:21031)
    at t._createInstance (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:4395)
    at t._createAndCacheServiceInstance (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:5370)
    at t._getOrCreateServiceInstance (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:4556)
    at Object.o.get (C:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:8:2679)

**Comment by shadowkyros** [comment]
Final Update:
I found a config file that I had missed when trying to uninstall/reinstall this program. It was located here: %APPDATA%\Code\User\settings.json. Inside of the config file was an outdated path to python (I was trying to get switch between python 2 and 3) that was causing the program not to open. I removed that path from the config file, saved it, and now my Visual Studio Code opens correctly.
